### ISSUE TITLE
Plasma 6 support

### ISSUE THREAD

**Issue by Electrenator** [doncsugar/Genome-Spectrum#7]
I would absolutely love if these themes would have support for usage within plasma 6 global themes. Especially spectrum-charoite which I quite like.

Not sure wha's different since the previous plasma version. Never looked at how themes actually work myself before.

**Comment by Electrenator** [comment]
Theme does seem to be available already under plasma 6. Just doesn't seem available as global theme there

**Comment by doncsugar** [comment]
Correct, you should be able to use the color scheme, plasma theme, and perhaps aurorae theme to get the full style. I am unsure of how the plasma theme's compatibility has changed with plasma 6.

**Comment by Electrenator** [comment]
> Correct, you should be able to use the color scheme, plasma theme, and perhaps aurorae theme to get the full style. I am unsure of how the plasma theme's compatibility has changed with plasma 6.

Not sure either but yes your right about that! Am able to use it, just not the global theme. Setting the parts individually does still work like a charm
